package.json

```diff
@@ -36,40 +36,42 @@
     "@sentry/node": "^6.0.4"
   },
   "optionalDependencies": {
-    "@nestjs/graphql": "^6.2.4 || ^7.0.0"
+    "@nestjs/graphql": "^8.0.2"
   },
   "peerDependencies": {
-    "@nestjs/common": "^7.0.0",
+    "@nestjs/common": "^8.0.0",
     "reflect-metadata": "^0.1.12",
-    "rxjs": "^6.4.0"
+    "rxjs": "^7.0.0"
   },
   "devDependencies": {
+    "@apollo/gateway": "^0.38.0",
     "@commitlint/cli": "13.1.0",
     "@commitlint/config-conventional": "13.1.0",
-    "@nestjs/common": "7.6.18",
-    "@nestjs/core": "7.6.18",
-    "@nestjs/platform-express": "7.6.18",
-    "@nestjs/platform-socket.io": "7.6.18",
-    "@nestjs/testing": "7.6.18",
-    "@nestjs/websockets": "7.6.18",
-    "@types/jest": "26.0.24",
-    "@types/node": "14.17.9",
-    "@types/socket.io-client": "1.4.36",
+    "@nestjs/common": "8.0.6",
+    "@nestjs/core": "8.0.6",
+    "@nestjs/platform-express": "8.0.6",
+    "@nestjs/platform-socket.io": "8.0.6",
+    "@nestjs/testing": "8.0.6",
+    "@nestjs/websockets": "8.0.6",
+    "@types/jest": "27.0.1",
+    "@types/node": "16.6.1",
     "@types/supertest": "2.0.11",
     "apollo-server-express": "2.25.2",
-    "apollo-server-testing": "2.25.2",
+    "class-transformer": "^0.4.0",
```

Comment:
I don't see this new dependencies used anywhere. Are they needed?

Reply:
```
npm WARN @nestjs/mapped-types@0.0.5 requires a peer of class-transformer@^0.2.3 but none is installed. You must install peer dependencies yourself.
npm WARN @nestjs/mapped-types@0.0.5 requires a peer of class-validator@^0.11.1 || ^0.12.0 but none is installed. You must install peer dependencies yourself.
```